packages/components/src/Tree/TreeStyle.tsx

```diff
@@ -58,37 +77,44 @@ export const TreeStyle = styled.div<TreeStyleProps>`
 
   & > ${Accordion} {
     & > ${AccordionContent} {
-      ${({ border, depth, theme }) =>
-        border && generateTreeBorder(depth, theme)}
+      ${({ border, depth, indicatorSize, theme }) =>
+        border && generateTreeBorder(depth, indicatorSize, theme)}
     }
 
     & > ${AccordionDisclosureStyle} {
+      ${ListItem} {
+        font-weight: ${({ branchFontWeight, theme: { fontWeights } }) =>
+          branchFontWeight ? fontWeights.normal : fontWeights.semiBold};
+      }
+
       ${listItemBackgroundColor}
       background-clip: padding-box;
       color: ${({ disabled, theme: { colors } }) =>
         disabled ? colors.text1 : colors.text5};
-      font-weight: ${({ branchFontWeight, theme: { fontWeights } }) =>
-        branchFontWeight ? fontWeights.normal : fontWeights.semiBold};
-      padding-right: ${({ theme }) => theme.space.xxsmall};
-      ${({ depth, theme }) => generateIndent(depth, theme)}
+      /**
+        Tree's padding-right is handled by the internal item
+       */
+      padding-right: 0;
+      ${({ depth, indicatorSize, theme }) =>
+        generateIndent(depth, indicatorSize, theme)}
     }
   }
 
   ${({ dividers }) => dividers && dividersCSS}
 
-  ${TreeItemInner} {
-    border-width: 0;
-    height: 100%;
-
-    & > ${TreeItemLabel} {
-      background-color: transparent;
-      padding-left: ${({ theme }) => theme.space.none};
-    }
+  & > ${Accordion} > ${AccordionContent} > ${List} > ${TreeHeading},
+  & > ${Accordion} > ${AccordionContent} > ${List} > ${ListItem} > button,
+  & > ${Accordion} > ${AccordionContent} > ${List} > ${ListItem} > a {
```

Comment:
Nesting could help make this easier to read:


```suggestion
  & > ${Accordion} > ${AccordionContent} > ${List} {
    & > ${TreeHeading}, & > ${ListItem} > button, ${ListItem} > a {
```

Reply:
Honestly I'm not sure my refactor here really helps - seems more complex if anything. AKA: Ignore me.